packages/flutter/lib/src/material/popup_menu.dart

```diff
@@ -92,8 +91,9 @@ class _PopupMenuItemState<T extends PopupMenuItem<dynamic>> extends State<T> {
 
     Widget item = new DefaultTextStyle(
       style: style,
-      child: new Baseline(
-        baseline: config.height - _kBaselineOffsetFromBottom,
+      child: new Align(
+        widthFactor: 1.0,
+        alignment: FractionalOffset.centerLeft,
```

Comment:
Yeah, that's why we did it this way originally.  The trouble is that the OEM widgets just center the text inside the item and we get bug reports about not matching.  I guess our philosophy is to follow the spec rather than following implementations, so I can put this back and mark the bug WONTFIX if you prefer.